### ISSUE TITLE
add Microsoft Windows support

### ISSUE THREAD

**Issue by aploium** [tflearn/tflearn#490]
Recently, tensorflow has just announced it's native windows support:
https://github.com/tensorflow/tensorflow/issues/17#issuecomment-263646199

Currently, tflearn has some compatible problems in windows
eg: 
```python
# helpers.trainer.Trainer line 65
tensorboard_dir="/tmp/tflearn_logs/"
```

Thank you very much.

**Comment by HWiese1980** [comment]
The first thing that I've noticed when trying to use the current stable (`pip install tflearn`) is that _curses is missing. afaik there's no `curses` library for Windows. Besides that I don't see what curses has got to do with deep learning and so why it's a requirement for tflearn. Thus, a first step might be to make everything curses related optional and completely drop it for the Windows port.

**Comment by benb969** [comment]
I was able to install curses from here:
http://www.lfd.uci.edu/~gohlke/pythonlibs/#curses
And was then able to use tflearn to some extent on windows, but got different issues (will post later)

**Comment by aploium** [comment]
@benb969 
I may have got the same problem as you, just as the #480 

> ValueError: No variables to save

and then I solved it by using this repo
https://github.com/tflearn/tflearn/issues/480#issuecomment-264417792

because only tensorflow v0.12 supports windows, for now

**Comment by aymericdamien** [comment]
Yes, we should start to work on a version compatible with Windows. We are currently working on some fix for TensorFlow 0.12. Anyone that wants to help with that is welcome :)

**Comment by HWiese1980** [comment]
Are you sure that `curses` was installed successfully? It should work if it did.

**Comment by edtruji** [comment]
Are you sure that curses was installed successfully?
In windows curses is installed using the .whl  but successfully I don't think so
Unicode characters are not supported. 

Running a example code
UnicodeDecodeError: 'ascii' codec can't decode byte 0xa8 in position 6:

**Comment by satadru5** [comment]
once going to use tflearn it is showing same issue..

ImportError: No module named '_curses'


Can u please help me out...

**Comment by gepolv** [comment]
Same issue here.

**Comment by asdf154** [comment]
this guy was able to run tflearn with GPU support on windows 10 - 64 bit:
[https://github.com/tflearn/tflearn/issues/539](https://github.com/tflearn/tflearn/issues/539)
